### ISSUE TITLE
Support FSSPEC backend

### ISSUE THREAD

**Issue by oeway** [imjoy-team/imjoy-elfinder#105]
fsspec have done great work on supporting all kinds of backends:
https://filesystem-spec.readthedocs.io/en/latest/api.html#built-in-implementations

We should take advantage of that.

**Comment by crackytsi** [comment]
Yes, would be nice to support also multiple-roots in parallel... ;)
